elyra/pipeline/resources/properties.json

```diff
@@ -67,7 +67,7 @@
           "default": "Filename"
         },
         "description": {
-          "default": "The path to the notebook file.",
+          "default": "The path to the Notebook file.",
```

Comment:
Nit: it should be lower case because it's not a name, place, ...

Reply:
Building on my comment above (re:changing the label `Python` back to `Python Script` and same for R), we can then easily change the case here by changing the generic_properties jinja template to render the lowercase version of the label instead. 

`{{ component.name|lower }}`